alchemy/src/cloudflare/vite-plugin.ts

```diff
@@ -0,0 +1,9 @@
+import { cloudflare, type PluginConfig } from "@cloudflare/vite-plugin";
+
+export const alchemyVitePlugin = (config?: PluginConfig) => {
+  return cloudflare({
+    configPath: ".alchemy/local/wrangler.jsonc",
+    persistState: { path: ".alchemy/miniflare" },
```

Comment:
this might not work when `Website.cwd` is set for monorepo use-cases

Reply:
Which one, the `configPath` or `persistState`?

If it's the `configPath`, isn't the wrangler.jsonc written to the target `cwd`?

For the `persistState`, see [here](https://github.com/sam-goodwin/alchemy/pull/758/files#r2238055581).

Reply:
both - any reference to .alchemy folder could be wrong, no?